conda_build/skeletons/cpan.py

```diff
@@ -471,12 +474,74 @@ def latest_pkg_version(pkg):
     except:
         pkg_list = None
     if pkg_list:
-        pkg_version = parse_version(pkg_list[-1].version)
+        pkg_version = LooseVersion(pkg_list[-1].version)
     else:
         pkg_version = None
     return pkg_version
 
 
+@memoized
```

Comment:
There is no ``memoized`` in Python ``2.7``, as we still support it we cannot use it

Reply:
It's made available from `conda_build.conda_interface` and used in many other places in cpan.py.  This use is reintroduction of `def core_module_version` from before PR #2156 becuase we need to use the Perl `corelist` utility as I explained in my commit message and the description of this PR.

If we need to clean these out of cpan.py, I'd like to do that in a separate PR that removes the import and all other uses.

Reply:
It's defined in conda.  I think it's fine here.